src/bootstrap/src/utils/helpers.rs

```diff
@@ -471,21 +472,63 @@ pub fn extract_beta_rev(version: &str) -> Option<String> {
     count
 }
 
+pub enum LldThreadMode {
+    SingleThreaded,
+    MultiThreaded,
+}
```

Comment:
I'd personally keep this a `bool`, but even with enum I'd select something less verbose, like `LldThreads::{Yes,No}` or something.

Reply:
Ok, renamed.